src/lib/liblow.c

```diff
@@ -33,7 +33,7 @@
 #include <sys/types.h>     /* socket() */
 #include <sys/socket.h>    /* socket() */
 #include <sys/un.h>        /* struct sockaddr_un */
-#include <sys/fcntl.h>     /* O_RDONLY */
+#include <fcntl.h>     /* O_RDONLY */
 #include <sys/stat.h>      /* stat() */
```

Comment:
Nit picking: Can you keep comment indent correct here?

Reply:
The alignment and ordering in liblow.c has been updatted.